### ISSUE TITLE
sct_register_multimodal: bad results

### ISSUE THREAD

**Issue by slevyrosetti** [neuropoly/spinalcordtoolbox#598]
The results were correct with the same data and the same command line one day ago. Maybe I miss something...

Data: at least two subjects available here: /Volumes/folder_shared/sct_issues/20151016_598
Command line:
~~~~
sct_register_multimodal -i ../t2/label/template/MNI-Poly-AMU_WM.nii.gz -d t2starcrop_wmseg_corrected.nii.gz -iseg ../t2/label/template/MNI-Poly-AMU_cord.nii.gz -dseg t2starcrop_seg_ok.nii.gz -p step=1,type=seg,algo=slicereg,metric=MeanSquares,iter=10,gradStep=0.5:step=2,type=im,algo=bsplinesyn,metric=MeanSquares,iter=15,gradStep=0.5:step=3,type=im,algo=syn,metric=MeanSquares,iter=5,gradStep=0.3
~~~~~
Results for d_sp_pain_04:

![screen shot 2015-10-16 at 3 47 46 pm](https://cloud.githubusercontent.com/assets/7769691/10552119/9890550c-741f-11e5-9078-fcbf3b3cf1e3.png)

Results for d_sp_pain_07:
![screen shot 2015-10-16 at 4 05 46 pm](https://cloud.githubusercontent.com/assets/7769691/10552153/d6df3d5a-741f-11e5-91bc-aa6c1a479d93.png)

**Comment by jcohenadad** [comment]
probably related to #596

**Comment by SaraDupont** [comment]
I fix (at least temporary)  #596, you should try now if the problem persist.
